.eslintrc

```diff
@@ -20,14 +20,17 @@
     "max-len": [
       "warn",
       {
-        "code": 150,
+        "code": 350,
         "tabWidth": 2,
         "ignoreComments": true
       }
     ],
-    "react/prop-types": "error",
+    "react/prop-types": "off",
     "react/jsx-filename-extension": [1, { "extensions": [".js", ".jsx"] }],
     "no-underscore-dangle": "off",
+    "no-console": "off",
+    "react/destructuring-assignment": [1, "never"],
```

Comment:
destructuring is something also improves readability, which is super important, there's also a lot of cool things you can do with it as well. it also introduces immutability which is good.
https://reactjs.org/tutorial/tutorial.html#why-immutability-is-important
https://medium.freecodecamp.org/the-basics-of-destructuring-props-in-react-a196696f5477
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment
I don't think we should change this rule.